lifecycle-operator/controllers/common/helperfunctions_test.go

```diff
@@ -525,3 +526,34 @@ func Test_GetEvaluationDefinition(t *testing.T) {
 		})
 	}
 }
+
+//nolint:dupl
+func TestGetControllerInfo(t *testing.T) {
+	// Test case 1: Request with name and namespace
+	req1 := ctrl.Request{
+		NamespacedName: types.NamespacedName{
+			Name:      "example",
+			Namespace: "test-namespace",
+		}}
+
+	info1 := GetControllerInfo(req1)
+	expected1 := map[string]string{
+		"name":      "example",
+		"namespace": "test-namespace",
+	}
+	require.Equal(t, expected1, info1)
+
+	// Test case 2: Request with empty name and namespace
+	req2 := ctrl.Request{
+		NamespacedName: types.NamespacedName{
+			Name:      "",
+			Namespace: "",
+		}}
+
+	info2 := GetControllerInfo(req2)
+	expected2 := map[string]string{
+		"name":      "",
+		"namespace": "",
+	}
+	require.Equal(t, expected2, info2)
```

Comment:
Let's remove this test case, as there is no possibility that request does not contain the name or the namespace of the object

Reply:
Alright I will do that